README.md

```diff
@@ -186,6 +186,12 @@ each 3 bytes of the id (with padding up to a number that divides by 4).
 E.g for 1, 2 or 3 bytes length there will be 4 characters in the id string,
 for 4, 5 or 6 there will be 8, etc.
 
+##### Session format
+
+It is possible to twick the generated session format using proper system property.
```

Comment:
_tweak_ the generated session _id_ format using proper configuration _parameter_.